setup.py

```diff
@@ -268,30 +270,12 @@ def run(self):
         "Intended Audience :: Science/Research",
         "License :: OSI Approved :: Apache Software License",
         "Operating System :: OS Independent",
-        "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.8",
-        "Programming Language :: Python :: 3.9",
         "Topic :: Scientific/Engineering :: Artificial Intelligence",
+        "Programming Language :: Python :: 3",
+    ]
+    + [
+        f"Programming Language :: Python :: 3.{i}"
+        for i in range(8, version_range_max)
     ],
     cmdclass={"deps_table_update": DepsTableUpdateCommand},
 )
-
-# Release checklist
```

Comment:
Lets keep the release checklist

Reply:
The bottom one looks like an abbreviated version. @sayakpaul, is it ok to remove this checklist or would you rather keep it?

Reply:
@sayakpaul, WDYT?

Reply:
OK, reversed.